src/codegen/codegen_c_visitor.cpp

```diff
@@ -4315,6 +4328,17 @@ void CodegenCVisitor::print_data_structures() {
     print_ion_var_structure();
 }
 
+void CodegenCVisitor::print_v_unused() const {
+    printer->add_line("#if NRN_PRCELLSTATE");
+    printer->add_line("inst->v_unused[id] = v;");
+    printer->add_line("#endif");
+}
+
+void CodegenCVisitor::print_g_unused() const {
+    printer->add_line("#if NRN_PRCELLSTATE");
+    printer->add_line("inst->g_unused[id] = v;");
```

Comment:
Should the right hand side have `g` instead of `v`?

Reply:
omg!